packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -140,6 +181,48 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     );
   }
 
+  private _editButton(sectionIndex: ComponentPageSection, url?: string): JSX.Element | undefined {
+    // Get section string for URLs and IDs.
+    const section = ComponentPageSection[sectionIndex];
+
+    // Generate edit URL from componentURL
+    let mdUrl: string | undefined = undefined;
+    if (this.props.componentUrl) {
+      mdUrl = `${this.props.componentUrl}/docs/${this.props.componentName}${section}.md`;
+    }
+
+    // Allow generated URL fallback.
+    const editUrl: string | undefined = url || mdUrl || undefined;
+
+    if (editUrl) {
+      // Get make section readable for tooltip. Add apostrophe to Don't
+      let readableSection = section;
+      if (sectionIndex === ComponentPageSection.Donts) {
+        readableSection = 'Don\'ts';
+      }
+      return (
+        <TooltipHost
+          content={ `Edit ${this.props.componentName} ${readableSection} on GitHub` }
```

Comment:
@betrue-final-final I am not sure how accessible this new button is. Is the tooltip with this generated content good enough?

Reply:
@betrue-final-final Just a friendly bump. I'll have Kirk Myhre take a look at this design-wise too.

Reply:
@betrue-final-final Are you just worried about the color of the pencil or the addition of the icon and its place in the tab order/narrator in general?

Reply:
I actually didn't look yet, but now that I am, I do have some.
1. The "View on Github" appearing as a pivot here is a usability issue actually. It looks like a pivot, but it's actually a link. That will affect everyone. 
2. Is it necessary to be able to edit all those things separately? Do we need that, or just enable editing the whole page? 

Let's answer these questions and then evaluate the resulting design for accessibility.

Reply:
Good comment on the "View on Github" link placement.  We'll have to figure out a better place for that within the content of the page.
Each one of those content "chunks" will be editable individually.  Based on the way Markdown works and the way we're crafting the community editing experience, we want to provide a quick, easy way for folks to update only the bits that they're responsible for without affecting any others.  make sense?

Reply:
1. It doesn't have to appear there, or even at all, but I thought it might be convenient.
1. It is necessary since they are each individual files and have to be that way to reduce non-dev confusion. If we are to implement more than three sections down the road it will be confusing to those not familiar with the component what file to edit if we link to just the /docs folder.

Reply:
Ok cool. However you might consider "Do's" and "Don'ts" as a single unit. I don't think one person will be assigned to "Do's" and another "Don'ts".

What color are the pencils?

Reply:
Right, ok that might be tricky on dos and don'ts. Just trying to simplify.

Reply:
Peter and I thought about how to combine Do's and Don'ts into one file. It'd require some sort of trigger in the markdown file for the markdown-to-jsx component to find and split into the required sections. It'd be too easy to break.

Reply:
@betrue-final-final The color of the pencils was `palette.neutralSecondary` but I'm thinking about removing any of the styles and just using IconButton's default styles.